app/utils/FeatureToggle.js

```diff
@@ -65,7 +66,7 @@ class FeatureToggle {
     static appwideToggles(req, ctx, appwideToggles) {
         if (appwideToggles.length) {
             ctx.featureToggles = {};
-            appwideToggles.forEach((toggleKey) => {
```

Comment:
this should be reverted, using native foreach is much faster

Reply:
Done